express-post-json/index.js

```diff
@@ -0,0 +1,27 @@
+var express = require('express');
+const app = express();
+var jsonExpress = express.json();
+var nextId = 1;
+var grades = {};
+
+app.use(jsonExpress);
+app.get('/api/grades', function (req, res) {
+  var gradesArr = [];
+  for (const i in grades) {
+    gradesArr.push(grades[i]);
+  }
+  res.json(gradesArr);
+});
+
+app.post('/api/grades', function (req, res) {
+  var reqBody = req.body;
+  reqBody.id = nextId;
+  nextId++;
+  res.status(201);
+  res.json(reqBody);
+});
```

Comment:
Be sure to store the data in your data model before responding so that future GET requests include the new data.